src/qt/trafficgraphwidget.cpp

```diff
@@ -11,18 +11,21 @@
 #include <QColor>
 #include <QTimer>
 
+#include <chrono>
 #include <cmath>
 
 #define DESIRED_SAMPLES         800
 
 #define XMARGIN                 10
 #define YMARGIN                 10
 
+using namespace std::chrono_literals;
+
 TrafficGraphWidget::TrafficGraphWidget(QWidget *parent) :
     QWidget(parent),
     timer(nullptr),
     fMax(0.0f),
-    nMins(0),
+    nMins(0min),
```

Comment:
Move into a member initializer?

Reply:
Addressed in 9bb48bf657c7f2d36309807fbc7b55d0ab581a76